composer.json

```diff
@@ -14,7 +14,7 @@
         "behat/mink-selenium2-driver": "1.4.x-dev as 1.3.x-dev",
         "composer/installers": "~1.5",
         "drupal-composer/drupal-scaffold": "~2.2",
-        "drupal/address": "^1.8",
+        "drupal/address": "^1.9",
```

Comment:
Same as the other PR, I think it's enough to fix the selector without updating the dependency.

Reply:
The selector would have the ["."](https://github.com/openeuropa/oe_translation/pull/118/files#diff-1a9ec38cbc32142ff1173d6e1fb95478c601d02f31d195a6eae628eaa0f5fb92R160) on 1.8 and doesn't have it on 1.9. I chose to fix the selector without the dot and to update the lowest version to 1.9 so the selector is always the same.

Reply:
Ah makes sense! It's also a dev requirement only... We can go like this, perfect.